### ISSUE TITLE
overscan

### ISSUE THREAD

**Issue by SquidPony** [yellowstonegames/SquidLib#6]
consider allowing an overscan option where the panel displays only a certain amount of the backing grid at any given time, with the ability to adjust the offset in the backing grid for display

should allow some sort of default backing to be displayed when the edge of the backing grid is within the displaying grid

**Comment by SquidPony** [comment]
I'm marking this as high priority since with the current libgdx implementation of panels it's manually intensive to create a viewport into a large map

**Comment by tommyettinger** [comment]
Agreed. I'll start on this after the 3.0.0-b7 release. The release should be tomorrow but the work on this may take a little while; I'm not sure just what will be involved and SquidPanel has become somewhat of a monstrous class.

**Comment by tommyettinger** [comment]
This is mostly implemented now. It uses libGDX's Camera to smoothly pan over cells as they come into view, and allows truly massive dungeon levels to be explored without generating extra parts mid-way through. The actual usage is somewhat cumbersome, and I think that it may be possible to streamline it soon (possibly using StarterKit to establish the initial objects this needs, since a Camera isn't normally used by SquidPanel). This looks like this, [Imgur Preview of panning Camera].(http://i.imgur.com/dkOdEjc.gifv)
